src/test/java/uk/gov/hmcts/reform/pip/data/management/errorhandling/exceptions/CustomExceptionsTest.java

```diff
@@ -39,6 +39,12 @@ void testCreationOfHearingNotFoundException() {
         assertEquals(TEST_MESSAGE, hearingNotFoundException.getMessage(), ASSERTION_MESSAGE);
     }
 
+    @Test
+    void testCreationOfHeaderValidationException() {
+        HeaderValidationException headerValidationException = new HeaderValidationException(TEST_MESSAGE);
+        assertEquals(TEST_MESSAGE, headerValidationException.getMessage(), ASSERTION_MESSAGE);
+    }
+
```

Comment:
will probably need to whack in the other custom validation exceptions just means we dont need anymore in the global handler but will still need the custom ones

Reply:
Added, cheers, didn't think of this.